package.json

```diff
@@ -45,91 +48,86 @@
     "lint-staged": "^12.5.0",
     "prettier": "^2.8.8",
-    "prettier-plugin-import-sort": "^0.0.7",
     "prettier-plugin-tailwindcss": "^0.4.1",
     "ts-loader": "9.4.4",
-    "turbo": "latest"
-  },
-  "resolutions": {
-    "@polkadot/apps-config": "0.132.1",
-    "@polkadot/api-augment": "11.2.1",
-    "@polkadot/api-base": "11.2.1",
-    "@polkadot/api-contract": "11.2.1",
-    "@polkadot/api-derive": "11.2.1",
-    "@polkadot/api": "11.2.1",
-    "@polkadot/rpc-augment": "11.2.1",
-    "@polkadot/rpc-core": "11.2.1",
-    "@polkadot/rpc-provider": "11.2.1",
-    "@polkadot/typegen": "11.2.1",
-    "@polkadot/types-augment": "11.2.1",
-    "@polkadot/types-codec": "11.2.1",
-    "@polkadot/types-create": "11.2.1",
-    "@polkadot/types-known": "11.2.1",
-    "@polkadot/types-support": "11.2.1",
-    "@polkadot/types": "11.2.1",
-    "@polkadot/hw-ledger-transports": "12.6.2",
-    "@polkadot/hw-ledger": "12.6.2",
-    "@polkadot/keyring": "12.6.2",
-    "@polkadot/networks": "12.6.2",
-    "@polkadot/util-crypto": "12.6.2",
-    "@polkadot/util": "12.6.2",
-    "@polkadot/x-bigint": "12.6.2",
-    "@polkadot/x-bundle": "12.6.2",
-    "@polkadot/x-fetch": "12.6.2",
-    "@polkadot/x-global": "12.6.2",
-    "@polkadot/x-randomvalues": "12.6.2",
-    "@polkadot/x-textdecoder": "12.6.2",
-    "@polkadot/x-textencoder": "12.6.2",
-    "@polkadot/x-ws": "12.6.2",
-    "@polkadot/example-react": "3.6.4",
-    "@polkadot/example-vue": "3.6.4",
-    "@polkadot/react-identicon": "3.6.4",
-    "@polkadot/react-qr": "3.6.4",
-    "@polkadot/reactnative-identicon": "3.6.4",
-    "@polkadot/ui-assets": "3.6.4",
-    "@polkadot/ui-keyring": "3.6.4",
-    "@polkadot/ui-settings": "3.6.4",
-    "@polkadot/ui-shared": "3.6.4",
-    "@polkadot/vue-identicon": "3.6.4",
-    "@polkadot/extension-base": "0.46.6",
-    "@polkadot/extension-chains": "0.46.6",
-    "@polkadot/extension-compat-metamask": "0.46.6",
-    "@polkadot/extension-dapp": "0.46.6",
-    "@polkadot/extension-inject": "0.46.6",
-    "@polkadot/extension-mocks": "0.46.6",
-    "@polkadot/extension-ui": "0.46.6",
-    "@polkadot/extension": "0.46.6",
-    "@polkadot/wasm-bridge": "7.3.2",
-    "@polkadot/wasm-crypto-asmjs": "7.3.2",
-    "@polkadot/wasm-crypto-init": "7.3.2",
-    "@polkadot/wasm-crypto-wasm": "7.3.2",
-    "@polkadot/wasm-crypto": "7.3.2",
-    "@polkadot/wasm-util": "7.3.2",
-    "@substrate/txwrapper-core": "7.2.0",
-    "@substrate/txwrapper-dev": "7.2.0",
-    "@substrate/txwrapper-examples": "7.2.0",
-    "@substrate/txwrapper-orml": "7.2.0",
-    "@substrate/txwrapper-polkadot": "7.2.0",
-    "@substrate/txwrapper-registry": "7.2.0",
-    "@substrate/txwrapper-substrate": "7.2.0",
-    "@substrate/txwrapper-template": "7.2.0",
-    "glob-parent": "^5.1.2",
-    "decode-uri-component": "0.2.1",
-    "semver": "~7.5.4"
+    "turbo": "1.3.1"
   },
-  "engines": {
-    "node": ">=18"
+  "pnpm": {
+    "onlyBuiltDependencies": [
+      "@sentry/cli"
+    ],
```

Comment:
I've tested this to 100% confirm that **only** dependencies listed here will have their postinstall scripts executed.

If we ditch `@sentry/cli`, or don't want to run *any* postinstall scripts, we can delete this section from the `package.json` and add [`ignore-scripts=true`](https://pnpm.io/npmrc#ignore-scripts) or [`ignore-dep-scripts=true`](https://pnpm.io/npmrc#ignore-dep-scripts) to `.npmrc`